enterprise/cypher/cypher-compiled-runtime-3.2/src/main/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/codegen/LogicalPlanConverter.scala

```diff
@@ -74,10 +75,16 @@ object LogicalPlanConverter {
     override val logicalPlan: LogicalPlan = singleRow
   }
 
-  private def hasLimit(p: LogicalPlan) = p.treeExists {
-    case _: plans.Limit => true
-    //top is limit + sort
-    case _: plans.Top => true
+  private def aggregationNotSupported(p: plans.Aggregation) = {
+    val notValid = p.treeExists {
+      case _: plans.Limit => true
+      //top is limit + sort
+      case _: plans.Top => true
+    }
+    notValid && p.aggregationExpression.values.toList.treeCount {
```

Comment:
I would have thought this should be `||` instead of `&&` so that either of the failing conditions can cause the compiled runtime to not be used.